src/v2/BaseLoginRouter.js

```diff
@@ -207,25 +217,34 @@ export default Router.extend({
         });
     }
 
-    // introspect stateToken when widget is bootstrap with state token
-    // and remove it from `settings` afterwards as IDX response always has
-    // state token (which will be set into AppState)
+    // Remove stateToken and other attributes to sanitize settings object.
+    // IDX response always has stateToken, that will be set into AppState)
+    const renderCallback = (idxResp) => {
```

Comment:
It seems more clear to have separated handler (success & error)

Reply:
It made my code at line 239 much easier to read.. also the handlers are just one time use.

Reply:
changed.